packages/gasket-plugin-lifecycle/index.js

```diff
@@ -26,8 +26,9 @@ async function resolve(root, name) {
     }
   }
 
+  const reg = /.*^((?!(\.test|\.spec).js).)*\.js$/;
   return files.filter(function filter(file) {
-    return path.extname(file) === '.js';
+    return !!reg.exec(file);
```

Comment:
You can use `.test` on a RegExp instead and avoid the `!!` (it'll work without the !! also)

Reply:
Thanks!